includes/Modules/Sign_In_With_Google.php

```diff
@@ -221,33 +392,54 @@ private function render_signin_button() {
 			return;
 		}
 
-		$redirect_url = add_query_arg( 'action', 'google_auth', wp_login_url() );
-		if ( substr( $redirect_url, 0, 5 ) !== 'https' ) {
+		$login_uri = add_query_arg( 'action', 'google_auth', wp_login_url() );
+		if ( substr( $login_uri, 0, 5 ) !== 'https' ) {
 			return;
 		}
 
+		$redirect_to = $this->context->input()->filter( INPUT_GET, 'redirect_to' );
+		$redirect_to = trim( $redirect_to );
+
 		// Render the Sign in with Google button and related inline styles.
 		?>
-<!-- <?php echo esc_html__( 'Sign in with Google button added by Site Kit', 'google-site-kit' ); ?> -->
+<!-- Sign in with Google button added by Site Kit -->
 <?php /* phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript */ ?>
 <script src="https://accounts.google.com/gsi/client"></script>
 <script>
 ( () => {
-	google.accounts.id.initialize({
+	const parent = document.createElement( 'div' );
+	document.getElementById( 'login' ).insertBefore( parent, document.getElementById( 'loginform' ) );
+
+	google.accounts.id.initialize( {
```

Comment:
Docs show this within an `onload` handler (likely due to async) but we should probably make sure it's there before calling in case the script is blocked from loading https://developers.google.com/identity/gsi/web/reference/js-reference#google.accounts.id.initialize

Reply:
This is not really needed. We load the script synchronously, so `google.accounts.id.*` will always be there.